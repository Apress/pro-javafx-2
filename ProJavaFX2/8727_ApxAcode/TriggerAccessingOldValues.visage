/*
 * @test
 * @run
 */

var i = 1024 on replace oldValue {
  println("Variable i changed from {oldValue} to {i}.");
}

i = 2048;
