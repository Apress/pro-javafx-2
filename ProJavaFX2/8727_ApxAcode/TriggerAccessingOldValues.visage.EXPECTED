Variable i changed from 0 to 1024.
Variable i changed from 1024 to 2048.
